optimus/df_transformer.py

```diff
@@ -749,63 +747,63 @@ def dateTransform(self, columns, currentFormat, outputFormat):
         :param  outputFormat    output date string format to be expected.
         """
         # Check if currentFormat argument a string datatype:
-        self.__assertTypeStr(currentFormat, "currentFormat")
+        self._assert_type_str(currentFormat, "currentFormat")
         # Check if outputFormat argument a string datatype:
-        self.__assertTypeStr(outputFormat, "outputFormat")
+        self._assert_type_str(outputFormat, "outputFormat")
         # Check if columns argument must be a string or list datatype:
-        self.__assertTypeStrOrList(columns, "columns")
+        self._assert_type_str_or_list(columns, "columns")
 
         if isinstance(columns, str):
             columns = [columns]
 
         # Check if columns to be process are in dataframe
-        self.__assertColsInDF(columnsProvided=columns, columnsDF=self.__df.columns)
+        self._assert_cols_in_df(columns_provided=columns, columns_df=self._df.columns)
 
         exprs = [date_format(unix_timestamp(c, currentFormat).cast("timestamp"), outputFormat).alias(
-            c) if c in columns else c for c in self.__df.columns]
+            c) if c in columns else c for c in self._df.columns]
 
-        self.__df = self.__df.select(*exprs)
+        self._df = self._df.select(*exprs)
 
-        self.__addTransformation()  # checkpoint in case
+        self._add_transformation()  # checkpoint in case
 
         return self
 
-    def ageCalculate(self, column, dateFormat, nameColAge):
+    def age_calculate(self, column, date_format, name_col_age):
```

Comment:
```
def age_calculate(self, column, dates_format, name_col_age):
        """
        This method compute the age of clients based on their born dates.
        :param  column      Name of the column born dates column.
        :param  date_format  String format date of the column provided.
        :param  name_col_age  Name of the new column, the new columns is the resulting column of ages.

        """
        # Check if column argument a string datatype:
        self._assert_type_str(column, "column")

        # Check if date_format argument a string datatype:
        self._assert_type_str(dates_format, "dates_format")

        # Asserting if column if in dataFrame:
        assert column in self._df.columns, "Error: Column assigned in column argument does not exist in dataFrame"

        # Output format date
        Format = "yyyy-MM-dd"  # Some SimpleDateFormat string

        exprs = format_number(
            mag(
                months_between(date_format(
                    unix_timestamp(column, dates_format).cast("timestamp"), Format), current_date()) / 12), 4).alias(
            name_col_age)

        self._df = self._df.withColumn(name_col_age, exprs)

        self._add_transformation()  # checkpoint in case

        return self
```